pkg/helmt/helmt.go

```diff
@@ -13,22 +15,28 @@ import (
 )
 
 var (
-	Output       = color.Output
-	Error        = color.Error
-	Execute      = execCommand
-	removeOutput = removeOutputCommand
+	Output                        = color.Output
+	Error                         = color.Error
+	execute                       = execCommand
+	removeOutput                  = removeOutputCommand
+	generateKustomizationFunction = generateKustomization
```

Comment:
Would be nice to align the naming with the other function pointers:
`generateKustomization = generateKustomizationCommand`